storage/plugins/mobile-api/src/Push/PushDispatcher.php

```
@@ -565,11 +575,13 @@ private function releaseClaim(int $userId, string $eventKey): void
     private function dueSoonDays(): int
     {
         try {
-            $days = (int) ((new \App\Models\SettingsRepository($this->db))->get('loans', 'reminder_days_before', '3') ?? 3);
+            $days = (int) ((new \App\Models\SettingsRepository($this->db))->get('advanced', 'days_before_expiry_warning', '3') ?? 3);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '578', column '33'). (undefined)

(MissingImport)